pkg/image/bootc_disk.go

```diff
@@ -56,6 +59,25 @@ func (img *BootcDiskImage) InstantiateManifestFromContainers(m *manifest.Manifes
 		qcow2Pipeline.Compat = img.Platform.GetQCOW2Compat()
 		qcow2Pipeline.SetFilename(img.Filename)
 		return qcow2Pipeline.Export(), nil
+	// TODO: refactor to share this with disk.go
+	case platform.FORMAT_VMDK:
+		vmdkPipeline := manifest.NewVMDK(buildPipeline, baseImage)
+		vmdkPipeline.SetFilename(img.Filename)
+		return vmdkPipeline.Export(), nil
+	case platform.FORMAT_OVA:
+		vmdkPipeline := manifest.NewVMDK(buildPipeline, baseImage)
+		ovfPipeline := manifest.NewOVF(buildPipeline, vmdkPipeline)
+		tarPipeline := manifest.NewTar(buildPipeline, ovfPipeline, "archive")
```

Comment:
Specifying `nil` instead of the `buildPipeline` as a first argument to each `manifest.New...()` makes them run outside the build pipeline, so on the host.  See above:
```go
		// qcow2 runs without a build pipeline directly from "bib"
		qcow2Pipeline := manifest.NewQCOW2(nil, baseImage)
```

Reply:
I get a Go nil pointer exception when I try that with `manifest.NewVMDK` because it always calls `buildPipeline.addDependent(p)`.  Though the docs...ah, I see now that I look at the equivalent code in `NewQCOW2`; adjusting.